docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,38 @@
+# Release 403 (9 Nov 2022)
+
+## General
+
+* Round `TimeWithTimeZone` values when `PARAMETRIC_DATETIME` is not set. ({issue}`14950`)
```

Comment:
This should be described in terms of user facing behavior. In particular, this is about fixing handling of variable precision time with time zone types for legacy clients that only understand precision = 3.